tests/test_markitdown.py

```diff
@@ -179,7 +194,7 @@ def test_markitdown_exiftool() -> None:
         assert target in result.text_content
 
 
-if __name__ == "__main__":
+if __name__main__":
```

Comment:
It looks like there's a small typo in the `if __name__main__":` line

Reply:
Thanks for your checking. I have already fix it.